wgpu-hal/src/metal/adapter.rs

```diff
@@ -850,20 +850,26 @@ impl super::PrivateCapabilities {
             } else {
                 Self::version_at_least(major, minor, 11, 0)
             },
+            //Depth clipping is supported on all macOS GPU families and iOS family 4 and later
+            supports_depth_clamping: device.supports_feature_set(MTLFeatureSet::iOS_GPUFamily4_v1)
+            || device.supports_feature_set(MTLFeatureSet::macOS_GPUFamily1_v1)
```

Comment:
this is really nitty, but we should use `os_is_mac` for consistency

Reply:
you mean instead of checking MTLFeatureSet::macOS_GPUFamily1_v1 featureset support?
```
supports_depth_clamping: device.supports_feature_set(MTLFeatureSet::iOS_GPUFamily4_v1)
             || os_is_mac
```
You're right, may be a bit confusing to do the same featureset check multiple times.

Reply:
yeah

Reply:
Done